cli/data.py

```diff
@@ -88,149 +57,21 @@ def main():
     help="Disable to skip loading datasets and instead re-use data from combined-raw.pkl.gz (much faster)",
     default=True,
 )
-@click.option("--state", type=str, help="For testing, a two letter state abbr")
-@click.option("--fips", type=str, help="For testing, a 5 digit county fips")
+@click.option(
+    "--states", "-s", type=str, multiple=True, help="For testing, two letter state abbrev's"
+)
 def update(
     aggregate_to_country: bool,
     print_stats: bool,
     refresh_datasets: bool,
-    state: Optional[str],
-    fips: Optional[str],
+    states: Optional[List[str]],
 ):
-    """Updates latest and timeseries datasets to the current checked out covid data public commit"""
-    path_prefix = dataset_utils.DATA_DIRECTORY.relative_to(dataset_utils.REPO_ROOT)
```

Comment:
Most of the below code has been moved to `libs/datasets/dataset_orchestrator.py`